### PR TITLE
fix (Fog) : re active fog in globeView

### PR DESCRIPTION
**Pull request by gchoqueux**

### CODE REVIEW

File: src/Core/Prefab/GlobeView.js
Comment (peppsac) [inline src/Core/Prefab/GlobeView.js]: Where do these formula come from?

File: src/Process/TiledNodeProcessing.js
Comment (peppsac) [inline src/Process/TiledNodeProcessing.js:129]: I'm not sure it's the best place to store `fogDistance`...
Maybe it would make sense in `Camera` but I have no strong opinion about this.

**Review by peppsac — COMMENTED** [review]
This `fog` feature is only active when log depth buffer is enabled:
```glsl
        #if defined(USE_LOGDEPTHBUF) && defined(USE_LOGDEPTHBUF_EXT)
            float depth = gl_FragDepthEXT / gl_FragCoord.w;
            float fogIntensity = 1.0/(exp(depth/distanceFog));
        #else
            float fogIntensity = 1.0;
        #endif
```
and for `GlobeView`.

Don't you think it would a good idea to fix these 2 limitations?

Last but not least: your PR lacks a before|after screenshot :)

File: src/Core/Prefab/GlobeView.js
Comment (gchoqueux) [reply src/Core/Prefab/GlobeView.js]: The old formula was deleted from code

**Comment by gchoqueux** [comment]
This PR restores a broken stuff
We will see to add other features

**Review by peppsac — APPROVED** [review]
You're restoring a broken feature but:
- it's still broken in PlanarMode so now it's half-broken. Is it really better?
- the used formula could used 1 or 2 lines of comments :-/
```js
fog = size * 160 * Math.pow((len - size * 0.99) * 0.25 / size, 1.5)
```

Anyway, feel free to merge this if you need it.

**Comment by peppsac** [comment]
@gchoqueux could you update this PR please? (I assume you still want to merge it?)

**Comment by peppsac** [comment]
Btw, this PR broke all planar modes examples.

**Comment by gchoqueux** [comment]
the problem is identified:  `planar.fogDistance` is undefined

now how fix it in PR #550

**Comment by autra** [comment]
Ok thanks.

When something is broken, I think we should adopt the following process:
- revert the faulty commit (this PR for instance)
- create a PR with the fixed commit *only* (don't mix it with features or other fix, like in #550)

That way, master (and our website) stays functional at all time, and we guarantee that the fix is merged quickly. Now, things are broken, and we have to wait until the feature is ready to get merged to have the fix. Not good!

**Pull request merged**
